lib/obfuscate_id.rb

```diff
@@ -34,8 +35,12 @@ def has_obfuscated_id?
       true
     end
 
-    def deobfuscate_id(obfuscated_id)
```

Comment:
There is also a an instance method that wraps this defined below. You'll need to update that one to us this new name, and potentially rename it as well.

It feels safest to maybe alias this with the old one, because who know who might be calling this static method somewhere